recirq/quantum_chess/quantum_board.py

```diff
@@ -631,7 +635,11 @@ def do_move(self, m: move.Move) -> int:
 
             # Find all the squares on both paths
             path_qubits = self.path_qubits(m.source, m.target)
+            if tqubit2 in path_qubits:
```

Comment:
Do you think we should fold this into the path_qubits() function (maybe have an optional exclude keyword argument?)

Reply:
I considered it, but I prefer how it is now where path_qubits corresponds to a single clear concept.